sirepo/job_driver/__init__.py

```diff
@@ -353,6 +370,7 @@ def init(job_supervisor_module):
     job_supervisor = job_supervisor_module
     cfg = pkconfig.init(
         modules=((_DEFAULT_MODULE,), set, 'available job driver modules'),
+        idle_check_secs=(1800, int, 'how many seconds to wait between checks'),
```

Comment:
parse_seconds. reuse name: idle_task_secs?

Reply:
parse_seconds  - https://github.com/radiasoft/sirepo/pull/2804/commits/697d75b03e5b4d2b980c6307ba4ed4910673df0d

What do you mean by "reuse name". Don't reuse the name from job_driver.docker?